specs/bls_signature.md

```diff
@@ -65,14 +67,16 @@ We require:
 
 ### `hash_to_G2`
 
+Note: The following function is an (insecure!) placeholder that will be replaced according to the recommendations of the ongoing BLS12-381 standardisation effort.
```

Comment:
Note for the note: the hash-and-check function is insecure since it is easy to waste verifiers' or signers' time by accidentally or adversarially choosing a difficult to hash message. Related #898, ia.cr/2019/403 .